File: week-04/day-01/sharpie.py
class Sharpie(object):
    def __init__(self, color, width):
        self.color = str(color)
        self.width = float(width)
        self.ink_amount = 100.0

    def use(self, amount):
        self.ink_amount -= amount
        return self.ink_amount

    def __repr__(self):
       return "Sharpie color: {}, Sharpie width: {}".format(self.color, self.width)

sharp = Sharpie("medve",50)
print(sharp.color)
sharp.use(6.5)
print(sharp.ink_amount)

File: week-08/day-01/sharpie.js
function Sharpie(color, width) {
    this.color = color;
    this.width = width;
    this.inkAmount = 100;
    this.use = function(){
        this.inkAmount -= this.width;
        if (this.inkAmount < 0) {
            this.inkAmount = 0;
        }
    };
}

let sharp = new Sharpie('red', 55);
while (sharp.inkAmount > 0) {
    sharp.use();
    console.log(sharp.inkAmount);
}